action.yml

```
@@ -1,6 +1,6 @@
 name: 'copy-issue-labels'
 description: 'Copy labels from linkend issues'
-author: 'Michal Vanko'
+author: 'Michal Vanko, Darshit Chanpura'
```

```suggestion
author: 'Michal Vanko'
```
I'm very thankful for every PR and suggestion. I hope this won't sound ungrateful, but I won't consider you an author. It wouldn't be fair to me, who came up with the idea and the need for this action, and also to other contributors who also contributed to the repository. You will still be displayed in this repository as a contributor.

---

np